src/smart-components/request/request-detail/stage.js

```diff
@@ -78,29 +79,31 @@ class Stage extends Component {
         <DataListCell>
           <span style={ { textTransform: 'capitalize' } } id={ `${item.id}-state` }>{ `${item.state}` } </span>
         </DataListCell>
-        <DataListCell>
+        { requestActive && <DataListCell>
           <Level>
             <LevelItem>
               <Link to={ `/requests/detail/${item.request_id}/approve` }>
                 <Button variant="link" aria-label="Approve Request">
-                  Approve
+                    Approve
```

Comment:
why are you changing the indentation from 2 to 4 here (and also below)?

Reply:
updated to 2